### ISSUE TITLE
Add support for providing your own Properties object

### ISSUE THREAD

**Issue by bbottema** [bbottema/simple-java-mail#61]
Simple Java Mail currently supports loading config properties either directly from file or else from an InputStream.

It should also be possible to trigger this configure procedure with a provided Properties source for more flexibility.

Not only allows this for more flexibility, you can now provide non-String values which are used as-is, rather than parsed to types appropriate for Simple Java Mail (booleans, ints ad TransportStrategy etc).

**Comment by bbottema** [comment]
Released in 4.1.3.

**Comment by bbottema** [comment]
Current 9.x usage (verified against 9.3.2):

```java
Properties settings = new Properties();
settings.setProperty("simplejavamail.transportstrategy", "SMTPS");
settings.setProperty("simplejavamail.smtp.host", "smtp.example.com");
settings.setProperty("simplejavamail.smtp.port", "465");

ConfigLoader.loadProperties(settings, true); // merge; false replaces
Mailer mailer = MailerBuilder.buildMailer();
```

The old `SMTP_SSL` value was renamed to `SMTPS` in 5.0. `ConfigLoader` sets process-wide defaults, so load them before building mailers. See the [5.0 migration note](https://www.simplejavamail.org/migration-notes-5.0.0.html#section-transportstrategy-renamed) and [current property reference](https://www.simplejavamail.org/configuration.html#section-config-properties).
